docs/include/conf/speaking-tips.rst

```diff
@@ -0,0 +1,89 @@
+Giving a Presentation at Write the Docs
+=======================================
+
+Are you giving a talk at this year’s Write the Docs? Great! Here are a
+few things that may help you plan your talk and your slides.
+
+Facts
+-----
+
+-  Your talk will be 30 minutes long.
+-  There will be no time for questions immediately following your talk,
+   plan to use the entire slot.
+-  We strongly recommend that you practice your talk, ideally in front
+   of at least one other person.
+-  We will have a lapel mic for you, so you don’t need to hold a microphone.
+-  We will provide HDMI, USB-C and MiniDisplay port adapters.
+-  The display ratio of the projector is 16:9
+
+Presentation
+------------
+
+**We do not be allow time after the presentations for questions**, we
+feel posing questions for the speaker is best done in small groups.
```

Comment:
s/feel/believe

"feel" is weaker here. We have reasons, we're not just emoting all over the place.